pkg/test/framework/components/istio/operator.go

```diff
@@ -404,29 +447,58 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster kube.Cluster,
 				"--set", "values.global.remotePilotAddress="+remoteIstiodAddress.IP.String(),
 				// Use the local Istiod for CA
 				"--set", "values.global.caAddress="+"istiod.istio-system.svc:15012")
+
+			if isCentralIstio(c.environment, cfg) {
+				installSettings = append(installSettings,
+					"--set", fmt.Sprintf("values.istiodRemote.injectionURL=https://%s:%d/inject", remoteIstiodAddress.IP.String(), 15017),
+					"--set", fmt.Sprintf("values.base.validationURL=https://%s:%d/validate", remoteIstiodAddress.IP.String(), 15017))
+
+				// base must be installed first in order to create istio-reader-service-account, otherwise create-remote-secret command will fail
+				baseSettings := make([]string, len(installSettings))
+				_ = copy(baseSettings, installSettings)
+				baseSettings = append(baseSettings,
+					"-f", filepath.Join(env.IstioSrc, "tests/integration/multicluster/centralistio/testdata/iop-remote-base.yaml"))
+				if err := applyManifest(c, baseSettings, istioCtl, cluster.Name()); err != nil {
+					return fmt.Errorf("failed to deploy centralIstiod base for cluster %v: %v", cluster.Name(), err)
+				}
+				// remote ingress gateway will not start unless the create-remote-secret command has run and created the istiod-ca-cert configmap
+				if err := configureDirectAPIServerAccess(c.ctx, c.environment, cfg); err != nil {
+					return fmt.Errorf("failed to create-remote-secret: %v", err)
+				}
+			}
 		}
 	}
+	return applyManifest(c, installSettings, istioCtl, cluster.Name())
+}
+
+func isCentralIstio(env *kube.Environment, cfg Config) bool {
+	if env.IsMulticluster() &&
+		(cfg.ControlPlaneValues != cfg.RemoteClusterValues && cfg.RemoteClusterValues != "") {
```

Comment:
This seems like it would be easy to trigger unintentionally. Why not make it an explicit setting in the config?

Reply:
switched to using `cfg.Values` to perform the check